python-sdk/tests_integration/sql/operators/data_validation/test_check_table.py

```diff
@@ -22,7 +21,6 @@
         {
             "database": Database.BIGQUERY,
             "file": File(path=str(CWD) + "/../../../data/homes_main.csv"),
-            "table": Table(conn_id="gcp_conn_project"),
```

Comment:
Why we don't need it now?

Reply:
This connection was only utilized in these two tests, which appear to be unnecessary. However, I'm unsure why we included it.